league/dashboard/views.py

```diff
@@ -39,24 +39,33 @@ def players():
 
 
 @csrf_protect.exempt
-@blueprint.route('/games/', methods=['GET', 'POST'])
-def games():
+@blueprint.route('/games/', methods=['GET'])
+def list_games():
+    """List all games."""
+    form = GameCreateForm(request.form, csrf_enabled=False)
+    games = Game.query.all()
+    return render_template('dashboard/games.html', games=games,
+                           game_create_form=form)
+
+
+@csrf_protect.exempt
```

Comment:
Yeahhhh this is minor tech debt. I have to go around and remove all these and update my tests >.<